Replicator/DBWorker.cc

```diff
@@ -974,11 +974,8 @@ namespace litecore { namespace repl {
     // Insert all the revisions queued for insertion, and sync the ones queued for syncing.
     void DBWorker::_insertRevisionsNow() {
         auto revs = _revsToInsert.pop();
-        if (!revs) {
-            // No insertions scheduled, only syncs, so just do those:
-            _markRevsSyncedNow();
```

Comment:
FYI, this isn't related to the bug fix, but I realized this call is obsolete. It used to be there because there was a single trigger for inserting and syncing revs.